lessons/Algebra2/logarithmic1-exploring-wealth-health/langs/en-us/pages/hypothesis-graph.adoc

```diff
@@ -0,0 +1,36 @@
+= Do People in Rich Countries Live Longer?
+++++
+<style>
+table { width: 300px !important; margin-left: 0.5in; }
+th { text-align: center !important; }
+</style>
+++++
+
+[.linkInstructions]
+--
+- *Median life expectancy* us a measure of what a "typical lifespan" is for people in each country
+- *Per-Capita GDP* is a measure of what a "typical income" is for people in each country (in thousands of US Dollars).
+
+A point at (75, 62) would represent a country where the average GDP is $75k/yr and the median life expectancy is 62 years old.
+--
+
+@n In 2016, the United States had a median life expectancy of *80.1 years*, and a per-capita GDP of $58,890. Plot this point on the graph.
+
+@n In the table below, fill in your best guess for the median life expectancy for people in a country with each per-capita GDP:
```

Comment:
@schanzer. As far as I can tell, we've given them zero basis for guessing median life expectancy based on per-capita-GDP. I, personally, feel overwhelmed and uncomfortable by the idea of filling in this table. Is there a good reason not to at least offer a second data point as a frame of reference!

Reply:
added to checklist in description